decentraland/explorer
# fix: Block Explore and Tutorial buttons + Hide scene blockers while in BiW editor

### What this PR includes?
- [x] **Block Explore and Tutorial buttons**: Players can activate the "Explore & Play" and "Tutorial" functionality while editing a land, which can result in unsaved or unpublished changes. This funciton should be deactivated while editing a land, greying out the icon.
- [x] **Hide scene blockers**: Players shouldn't see any green blocker while ther're editing a land.

### Testing
1. Go to: https://explorer.decentraland.zone/branch/fix/Block-Explore-and-Tutorial-buttons-from-the-taskbar-while-in-BiW/index.html?ENV=zone&ENABLE_BUILDER_IN_WORLD&position=100,100&realm=fenrir-amber
2. Activate the Builder In World editor pressing the "**K**" key.
3. See the new loading screen.
